cpp/ql/test/library-tests/dataflow/dataflow-tests/test.cpp

```diff
@@ -594,7 +594,7 @@ void test_indirect_flow_to_array() {
   int* p = indirect_source();
   int* xs[2];
   xs[0] = p;
-  sink(*xs[0]); // $ ir=594:12 MISSING: ast SPURIOUS: ir=595:8
+  sink(*xs[0]); // $ ir MISSING: ast
```

Comment:
I still find this hard to follow. Which flow remains here? `ir=594:12` or `ir=595:8`? I actually suspect `ir=595:8`?

Reply:
Yeah, I agree. It's kinda difficult to see what's going on here when we cannot label the single source. I'll check this and write out which source we're actually catching here.

Reply:
FWIW, we're actually keeping the `ir=594:12` result 🎉 
<img width="878" alt="Screenshot 2023-03-08 at 09 04 27" src="https://user-images.githubusercontent.com/4527323/223669998-3e18bcbb-e7a0-4a15-86e8-3200a47e588b.png">

Reply:
I've clarified the result in https://github.com/github/codeql/pull/12432/commits/263b2082828a36b9a70d415eb0b0ce0b22adb136. Obviously, it would be nice to have an enforced annotation to capture this, but that's not possible (without introducing another source of flow that hits the sink).